packages/rancher-wins-upgrader/charts/scripts/upgrade.ps1

```diff
@@ -0,0 +1,44 @@
+$ErrorActionPreference = 'Stop'
+
+function Create-Directory
+{
+    param (
+        [parameter(Mandatory = $false, ValueFromPipeline = $true)] [string]$Path
+    )
+
+    if (Test-Path -Path $Path) {
+        if (-not (Test-Path -Path $Path -PathType Container)) {
+            # clean the same path file
+            Remove-Item -Recurse -Force -Path $Path -ErrorAction Ignore | Out-Null
+        }
+
+        return
+    }
+
+    New-Item -Force -ItemType Directory -Path $Path | Out-Null
+}
+
+function Transfer-File
+{
+    param (
+        [parameter(Mandatory = $true)] [string]$Src,
+        [parameter(Mandatory = $true)] [string]$Dst
+    )
+
+    if (Test-Path -PathType leaf -Path $Dst) {
+        $dstHasher = Get-FileHash -Path $Dst
+        $srcHasher = Get-FileHash -Path $Src
+        if ($dstHasher.Hash -eq $srcHasher.Hash) {
+            return
+        }
+    }
+
+    $null = Copy-Item -Force -Path $Src -Destination $Dst
+}
+
+Create-Directory -Path "c:\host\etc\rancher\wins"
+Transfer-File -Src "c:\scripts\config" -Dst "c:\host\etc\rancher\wins\config"
+Transfer-File -Src "c:\Windows\wins.exe" -Dst "c:\host\etc\rancher\wins\wins.exe"
```

Comment:
`c:\Windows/wins.exe` will need to be modified based on the path within the Docker image that should be created for this based on discussion in https://github.com/rancher/charts/pull/996/files#r575614057.

Currently, this is set based on the location of wins.exe provided in https://github.com/rancher/wmi_exporter-package/blob/8e0c309b986cfbfb96ec1dba0a40df612f149f36/package/windows/Dockerfile#L33.